plugins/captions-selector/templates/captionsSettingsTemplate.html

```diff
@@ -0,0 +1,13 @@
+<tp-yt-paper-icon-button class="player-captions-button style-scope ytmusic-player" icon="yt-icons:subtitles"
```

Comment:
note on naming: it's generally advised to use non-capital letters for file names (eg `captions-settings-template.html`) to avoid bugs on case insensitive OSes.